src/content/changes/highlights/2021-07-16.md

```diff
@@ -0,0 +1,17 @@
+---
+date: 2021-07-16T10:00:00
+title: Highlights for the week ending July 16, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Managed apps
+
+[NGINX IC v2.0.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v2.0.0/) is upgraded to controller container image v0.48.1. It contains performance improvements related to the admission webhook. **Breaking**: For security, it disables forwarding traffic to ExternalNameServices by default. To enable, view [changelog](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v2.0.0/) for details.
+
+[Prometheus Operator v0.9.0](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.9.0/) uses `giantswarm/config` to generate managed configuration. This is part of a larger [story](https://github.com/giantswarm/roadmap/issues/121) to automate the configuration management of installations.
```

Comment:
@kubasobon or @snizhana-dynnyk , @ubergesundheit , can you check if this is correct? :)

Reply:
Looks good :)

Reply:
thanks!